apps/etterlatte-oppdater-behandling/src/main/kotlin/MigreringHendelser.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.etterlatte
+
+import com.fasterxml.jackson.module.kotlin.treeToValue
+import no.nav.etterlatte.libs.common.logging.withLogContext
+import no.nav.etterlatte.libs.common.objectMapper
+import no.nav.etterlatte.libs.common.rapidsandrivers.correlationId
+import no.nav.etterlatte.libs.common.rapidsandrivers.eventName
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import no.nav.helse.rapids_rivers.River
+import org.slf4j.LoggerFactory
+import rapidsandrivers.BEHANDLING_ID_KEY
+import rapidsandrivers.HENDELSE_DATA_KEY
+import rapidsandrivers.behandlingId
+import rapidsandrivers.migrering.MigreringRequest
+import rapidsandrivers.migrering.Migreringshendelser.START_MIGRERING
+import rapidsandrivers.migrering.Migreringshendelser.VILKAARSVURDER
+import rapidsandrivers.withFeilhaandtering
+
+internal class MigreringHendelser(rapidsConnection: RapidsConnection, private val behandlinger: BehandlingService) :
+    River.PacketListener {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    init {
+        logger.info("initierer rapid for migreringshendelser")
+        River(rapidsConnection).apply {
+            eventName(START_MIGRERING)
+
+            correlationId()
+            validate { it.rejectKey(BEHANDLING_ID_KEY) }
+            validate { it.requireKey(HENDELSE_DATA_KEY) }
+        }.register(this)
+    }
+
+    override fun onPacket(packet: JsonMessage, context: MessageContext) {
+        withLogContext(packet.correlationId) {
+            withFeilhaandtering(packet, context, START_MIGRERING) {
+                logger.info("Mottatt migreringshendelse")
+
+                val hendelse: MigreringRequest = objectMapper.treeToValue(packet[HENDELSE_DATA_KEY])
+                val behandlingId = behandlinger.migrer(hendelse)
+                packet.behandlingId = behandlingId
```

Comment:
Ser at sakId sendes med for regulering, er det mer aktuelt der enn for migrering?

Reply:
Usikker, det kan godt vera det viser seg at vi treng det her også. For regulering trong vi det (viss eg hugsar rett) for å finne ut om ei yting var løpande på ein gitt dato, og det kan jo ha vore for ei heilt anna behandling enn denforrige.